src/picknik_ur_base_config/objectives/get_door_handle_config.yaml

```diff
@@ -0,0 +1,5 @@
+GetDoorHandle:
```

Comment:
PR https://github.com/PickNikRobotics/moveit_studio/pull/2974 looks to be partially merged in here, making this file a duplicate / unused. Could we remove this file and the `setup_mtc_open_lever_handle_door_config.yaml` files entirely?

Reply:
Good catch! Removed.